### PR TITLE
PM-11488: Add a switch to the autofill settings UI for enabling the accessibility service

### PR DESCRIPTION
**Pull request by david-livefront**
## 🎟️ Tracking

PM-11488

## 📔 Objective

This PR adds a switch to the autofill settings UI for enabling the accessibility service.

## 📸 Screenshots

<video src="https://github.com/user-attachments/assets/f03d5088-7ffb-44fe-b6e2-1833152a4306" width="350" />

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: app/src/main/java/com/x8bit/bitwarden/data/autofill/accessibility/manager/AccessibilityActivityManagerImpl.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import android.provider.Settings
+import androidx.lifecycle.LifecycleCoroutineScope
+import com.x8bit.bitwarden.data.platform.manager.AppForegroundManager
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+
+/**
+ * THe accessibility service name as it is known in the android manifest.
+ */
+private const val ACCESSIBILITY_SERVICE_NAME =
```
Comment (dseverns-livefront) [inline app/src/main/java/com/x8bit/bitwarden/data/autofill/accessibility/manager/AccessibilityActivityManagerImpl.kt:13]: ❓  and the answer may just be, this is how we want to treat string constants. Could we have this as a public constant in `BitwardenAppComponentFactory` or maybe have some `object BitwardenAppComponents { const val accessibiltityServiceName = "com.x8bit.yada.yada.yada" `

**Review by dseverns-livefront — DISMISSED** [review]
One comment that is non blocking for sure, just exposes my own personal for non-DRY string constants.

File: app/src/main/java/com/x8bit/bitwarden/data/autofill/accessibility/manager/AccessibilityActivityManagerImpl.kt (hunk 1 above)
Comment (david-livefront) [reply app/src/main/java/com/x8bit/bitwarden/data/autofill/accessibility/manager/AccessibilityActivityManagerImpl.kt]: That seems like a good idea, I'd like to make a separate PR where we can do that in a consistent way for all of those legacy app component names.

Comment (dseverns-livefront) [reply app/src/main/java/com/x8bit/bitwarden/data/autofill/accessibility/manager/AccessibilityActivityManagerImpl.kt]: Sounds good!

**Review by dseverns-livefront — DISMISSED** [review]

**Review by dseverns-livefront — APPROVED** [review]

**Comment by david-livefront** [comment]
Thanks @dseverns-livefront

**Pull request merged**